### ISSUE TITLE
Bar Series doesn't respect props passed as part of Data

### ISSUE THREAD

**Issue by mihirsoni** [uber/react-vis#1011]
From documentation it looks like BarSerie takes ```stroke (optional), fill (optional)``` props but it looks like it is not being passed down to actual SVG.

This can be reproduced here : https://codesandbox.io/s/6jy4py4w6w

Please let me know if I have misunderstood the doc.

**Comment by mcnuttandrew** [comment]
I totally agree the docs might be out of sorts. You can pass style props by giving BarSeries a style prop  like `<HorizontalBarSeries style={{fill: 'none}} />`. For more info check the code itself: https://github.com/uber/react-vis/blob/master/src/plot/series/bar-series.js 

Also, if you see problems with the docs, I would be delighted to accept a PR addressing your concerns!

**Comment by mihirsoni** [comment]
@mcnuttandrew  Thanks for response.

From what I see, it makes sense to be able able to accept it from each data point, I thought it would give more flexibility to the how bar is being rendered. 

For example if HorizontalBarSeries graph is being plotted using `x0` and if it will everyone  flexibility to change that bar style ?
